pyramid_oereb/contrib/data_sources/standard/sources/document.py

```diff
@@ -9,6 +9,9 @@ class DatabaseSource(BaseDatabaseSource, DocumentBaseSource):
     def read(self, office_records):
         """
         Central method to read all theme entries.
+
+        Args:
+            office_records (list): list of office records
```

Comment:
Can we use the terminology `(list of ...)` here, ... is the python path to the list element? Its more precise. We have several examples for that